api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
+<ul>
+    <li>Städa alltid upp efter dig. Glöm inte att fler medlemmar kommer senare för att använda samma ytor och maskiner. Den medlemmen kan vara du.</li>
+    <li>Se till att använda utrustning och lokaler på ett korrekt sätt. Om utrustning används på felaktivt vis kan du skada utrustningen och göra den omöjlig att använda för fler medlemmar. Den medlemmen kan vara du.</li>
+    <li>Informera om något är trasigt, försvunnet eller om det behöver fyllas på med något. Saker går sönder ibland, det är sånt som händer. Om du inte kan laga det själv så skriv på Slack. Om du inte gör det kommer utrustningen förbli trasig när fler medlemmar vill använda den. Den medlemmen kan vara du.</li>
+    <li>Ta inte utrustning med dig. Om du tar hem saker är den inte tillgänglig för fler medlemmar. Den medlemmen kan vara du.</li>
+</ul>
 
-<h3>Hur funkar styrelsen?</h3>
-<p>Styrelsen arbetar ideellt och möts ungefär 1 gång i månaden. Då går man igenom ekonomi, kommande medlemsintroduktions-datum, eventuella problem och tar beslut kring större inköp. Om du vill kontakta styrelsen så emaila <a href="mailto:info@makerspace.se">info@makerspace.se</a>.</p>
+<p>Ta ansvar för...</p>
+<ul>
+    <li>...dina projekt och ditt material. Om du lämnar projekt eller material utan att märka upp det så kommer det slängas.</li>
+    <li>...dina gäster. Du får ta in gäster för att visa lokalerna men de får inte arbeta på dina projekt. Barn under 16 år får vara med och kan arbeta på projekt under ditt ansvar.</li>
+    <li>...din säkerhet. Du ansvarar för din fysiska säkerhet i föreningens lokaler.</li>
+    <li>...din användning av maskiner och utrustning. Du har ansvar för att du använder dessa på ett säkert sätt. Läs alltid de tillhörande skyltarna och informationen på wikin innan du sätter igång.</li>
+    <li>...att följa vår nolltolerans mot brott. Avsteg leder till påföljder.</li>
+    <li>...att följa vår nolltolerans mot alkohol. Avsteg leder till påföljder.</li>
+    <li>...att inte bedriva kommersiell verksamhet. Det är okej att bygga prototyper för kommersiellt syfte, men det är förbjudet att bedriva tillverkning.</li>
+</ul>
+<p>För mer information om reglerna när du besöker makerspacet, läs wikisidan med våra <a href="https://wiki.makerspace.se/Lokalregler">Lokalregler</a>.</p>
 
+<h3>Städning, underhåll, ordning och reda:</h3>
+<ul>
+    <li>Pågående projekt, material och förvaringslådor kräver alla en förvaringslokal. Du kan skriva ut dessa i våra Memberbooth. Lär mer om dem på <a href="https://wiki.makerspace.se/Memberbooth">Memberbooth wikisida</a>.</li>
+    <li>Alla medlemmar har rätt att ha en förvaringslåda på makerspacet. Du hittar information om dessa på <a href="https://wiki.makerspace.se/Medlemsf%C3%B6rvaring">wikisidan för medlemsförvaring</a>.</li>
+    <li>Om du ser en full soptunna så töm soporna, släng påsen i soprummet och sätt in en ny påse. Så här går det till: <a href="https://wiki.makerspace.se/St%C3%A4drutiner">Städrutiner</a>.</li>
```

Comment:
```suggestion
    <li>Om du ser en full soptunna så töm soporna, släng påsen i soprummet och sätt i en ny påse. Så här går det till: <a href="https://wiki.makerspace.se/St%C3%A4drutiner">Städrutiner</a>.</li>
```

And perhaps instead of `Så här går det till:` it could say `Här finns instruktioner hur du hittar dit`.